### ISSUE TITLE
Error: "The item has been moved or deleted."

### ISSUE THREAD

**Issue by k4j8** [KyleWJohnston/outlook-move-to-thread#8]
Steps to reproduce:

1. Select two items in a conversation.
2. Run the macro.
3. One of the items gets moved to the correct folder, the other item remains in the existing folder, and the following error appears:
![2021-06-08 15_17_40-Microsoft Visual Basic](https://user-images.githubusercontent.com/7908653/121251895-fce7b980-c86c-11eb-8fb4-84e4fd1b1ff3.png)

**Comment by joffer77** [comment]
Did you ever solve this?

**Comment by k4j8** [comment]
No, but clicking "End" and then retrying works. If I make sure to only have one email selected before running the macro, I don't run into this issue.

**Comment by joffer77** [comment]
I get the error if I have opened one email and run the macro from a button I added. I see that clicking 'end' works to get the error to go away, but for me it seems the email is moved (correctly) before the error shows, and then fails showing the form/info window, which is the error not finding the email after its been moved I think.. It's too complicated code for me to troubleshoot, at least since it works in general, just have to click 'end' (or alt+e)
